deployments/datahub/config/common.yaml

```diff
@@ -28,6 +28,10 @@ jupyterhub:
         2019-fall-31002: {}
         # data 88
         2019-fall-33469: {}
+        # astr 128/256
+        2020-spring-astron-128-001-lab-001:
```

Comment:
@yuvipanda it should actually be `2020-spring-24268`, based on the "Class #" in the listing.

Reply:
@ryanlovett aaah, thanks. How about now?

Reply:
Looks good!